src/front/DataInspector.js

```diff
@@ -2,7 +2,18 @@ L.TileLayer.XRay = L.TileLayer.extend({
 
     getTileUrl: function (tilePoint) {
         this.options.version = Date.now();
-        this.options.showLayer = L.K.Config.dataInspectorLayer || '';
+        var keys = Object.keys(L.K.Config);
+        if (L.K.Config['dataInspectorLayer___all__'] === true) {
```

Comment:
Better to keep only one entry key in the Config object, something like `L.K.Config.dataInspector = {layerX: true, 'ayerY: false...}`